test_libs/pyspec/eth2spec/utils/ssz/ssz_typing.py

```diff
@@ -452,15 +276,14 @@ def is_list_type(typ):
     """
     Check if the given type is a list.
     """
-    return get_origin(typ) is List or get_origin(typ) is list
+    return isinstance(typ, type) and issubclass(typ, List)
 
 
 def is_bytes_type(typ):
     """
     Check if the given type is a ``bytes``.
     """
-    # Do not accept subclasses of bytes here, to avoid confusion with BytesN
-    return typ == bytes
+    return isinstance(typ, type) and issubclass(typ, Bytes)
```

Comment:
now that everything is a type, can we not do `issubclass(typ, Bytes)` directly in the implementation? (or alias that subclass check, if you find it ugly for readability. I think that's better than maintaining 10 `is_foobar_type` methods. Yay for consistent typing :+1: